test/support/page_objects.js

```diff
@@ -0,0 +1,74 @@
+
+import Common from './pages/common.js';
+import ConsolePage from './pages/console_page.js';
+import DashboardPage from './pages/dashboard_page.js';
+import DiscoverPage from './pages/discover_page.js';
+import HeaderPage from './pages/header_page.js';
+import SettingsPage from './pages/settings_page.js';
+import ShieldPage from './pages/shield_page.js';
+import VisualizePage from './pages/visualize_page.js';
+
+const common = new Common();
+const consolePage = new ConsolePage();
+const dashboardPage = new DashboardPage();
+const discoverPage = new DiscoverPage();
+const headerPage = new HeaderPage();
+const settingsPage = new SettingsPage();
+const shieldPage = new ShieldPage();
+const visualizePage = new VisualizePage();
+
+export default {
```

Comment:
When I see certain patterns in code I make certain assumptions. I don't think that everyone makes the same assumptions, but I imagine that most people familiar with a certain coding style make _similar_ assumptions.

One of those patterns is the plain object definition with function properties. When I see that pattern I assume that a collection of functions are being defined to provide related functionality, that those functions will likely access some shared variables, and that the object will be passed to or required by some other part of the code.

When I saw this object definition I started off with that assumption, but once I noticed the use of the `this` keyword I had to stop, rewind and reread the code while tracking what additional context  `this` provided.

When I read a class definition I automatically track the meaning of `this`, and the values that is provides, so there isn't a moment where I have to stop, rewind, and reread after discovering its use in the code.

Make sense?